app/assets/javascripts/components/stories/Sprint.js

```diff
@@ -83,8 +84,8 @@ class Sprint extends Component {
           onClick={this.onHeaderClick}
           isClosed={isClosed}
         />
-        <div className={`Sprint__body ${closedStyle}`}>
-          {stories && <Stories stories={stories} />}
+        <div data-cy={ column ? column : 'done' } className={`Sprint__body ${closedStyle}`}>
```

Comment:
What is the purpose of this if? Check if `column` has value? If so, you can rewrite like this:
```suggestion
        <div data-cy={column || 'done'} className={`Sprint__body ${closedStyle}`}>
```

Reply:
Also, I would extract this logic from here and put it at line 74.
Why? to keep your render method as clear as possible without any _visible_ logic, at least, you _HTML_.

It's easy to read 
```js
render (
  <Form onSubmit={submitHandler}>
    ...
  </Form>
)
```
 instead 
```js
render (
  <Form onSubmit={() => {
    // a
    // lot
    // of
    // logic
    // here
  }}>
    ...
  </Form>
)
```